setup.cfg

```diff
@@ -22,10 +20,12 @@ classifiers =
     Topic :: Scientific/Engineering :: Physics
 
 [options]
+# We set packages to find: to automatically find all sub-packages
+packages = find:
 requires = numpy
 zip_safe = False
 tests_require = pytest-astropy
-setup_requires = numpy>=1.16
+setup_requires = setuptools_scm
```

Comment:
Is `setup_requires` still necessary?
```suggestion
```

Reply:
We could remove it, but leaving it allows commands like ``python setup.py --version`` to still work, which can be useful in some cases.

Reply:
I didn't even know `setup.py --version` existed. Okay.